### PR TITLE
Countdown Goodie v2

### LINKED ISSUE #3500: Countdown: Needs to load alarm sound using versioned asset path
In production, all paths for `/share` are versioned based on the current repo release number.

For Goodies, we need to manually build the `/share` path by passing the current Goodie Repo version to the JS and construction the path.

An example of grabbing the Perl version number can be seen in the [SolarSystem](https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/lib/DDG/Goodie/SolarSystem.pm#L14) Goodie.

We need to use that same approach to ensure the audio asset is successfully loaded.

---

IA Page:  http://duck.co/ia/view/countdown 
[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @rasikapohankar

### PR DESCRIPTION
**Pull request by rasikapohankar**
###### What does your Pull Request do (check all that apply)?
- [x] Improvement
  - [x] Enhancement: **`{Countdown}: {Update javascript and implement design changes}`**
###### Description of changes

Changed the javascript code to replace updating of countdown values using `html()` with `meta.rerender`, used `DDH_async_add` to load moment.js, made changes to implement the design suggested by the design team.
In continuation of #2808 
Fix #3500 
###### People to notify (@mention interested parties)

@moollaza 

---

Instant Answer Page: https://duck.co/ia/view/countdown

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @rasikapohankar

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by edgesince84** [comment]
Should "countdown to tomorrow" be a 24-hour countdown?  It's near noon here, so I was expecting  ~twelve-hour countdown.

File: share/goodie/countdown/countdown.css [hunk 1]
```diff
@@ -1,19 +1,17 @@
...
-    vertical-align: top;
+.zci--countdown .number {    
+    font-size: 2.5rem;  /*for 36px*/
 }
 
-.zci--countdown .number,
-.zci--countdown .separator {
-    font-size: 2.5em;
+.zci--countdown .c-base__sub {
+    color: #999999;
```
Comment (moollaza) [inline share/goodie/countdown/countdown.css]: Ideally this should be applies with `meta.elClass` but the detail templates don't support that yet.

Instead we should add `tx-clr--grey` via JS. This way on the other themes the colour will change accordingly

**Comment by rasikapohankar** [comment]
Hey @moollaza, I have made the change for the `share` path (#3500) in this pull request. The check has failed because I haven't updated the test file to add the `goodie_version`. What number should be specified for `goodie_version` in the test file as it will keep changing? I checked `SolarSystem.t`, it has ignored all data.

**Comment by rasikapohankar** [comment]
Hi @moollaza, does this look okay?

**Comment by rasikapohankar** [comment]
@moollaza The conflict is due to the image which is not used anymore, so when this is merged the image will be deleted. Is this the proper way to do it or it should be restored here and deleted later on?

**Comment by mintsoft** [comment]
@rasikapohankar If this PR removes the use of the image, it should be deleted here. I.e. when this merge occurs then it should be deleted

**Comment by rasikapohankar** [comment]
@mintsoft The current version in the master also does not use the image but I hadn't removed it before..

**Comment by mintsoft** [comment]
@rasikapohankar That's fine, if it's unused it should be deleted; it's perfectly OK to bomb it out in this PR

**Comment by rasikapohankar** [comment]
Hi @moollaza @mintsoft, I broke this PR. I will open another one with the correct commits.

**Pull request closed without merging**